test/jit/test_peephole.py

```diff
@@ -217,3 +217,12 @@ def forward(self, x):
             self.run_pass("inline", conv_dim.graph)
             self.run_pass("peephole", conv_dim.graph)
             FileCheck().check("conv").check("dim").run(conv_dim.graph)
+
+    def test_normalized_is_op(self):
+        print("test_start")
+        @torch.jit.script
+        def convertible_is_op(x:bool, y:bool):
+            return x is True, False is x, x is y
+
+        FileCheck().check_count("aten::eq", 3, exactly=True).run(convertible_is_op.graph)
+        FileCheck().check_count("aten::__is__", 0, exactly=True).run(convertible_is_op.graph)
```

Comment:
Why don't we run it just to check that we did the right transformation ? You can manually run it yourself, and we also have a `checkScript` utility that runs a python function for example inputs and checks that the compiled version of that function gives the same results.